datadog/src/main/scala/zio/metrics/connectors/datadog/DatadogEncoder.scala

```diff
@@ -41,5 +41,8 @@ case object DatadogEncoder {
     (key, values) => Chunk.fromArray(withContainerId(key, values).toString().getBytes())
   }
 
+  private def makeStatsdEncoder(config: DatadogConfig): StatsdEncoder =
+    StatsdEncoder(config.entityId.map(eid => s"dd.internal.entity_id:$eid"))
```

Comment:
why not handle container ID here too?

Reply:
It's not a tag actually, it's a separate part of the datagram.

<img width="725" alt="Screen Shot 2023-11-21 at 5 57 45 PM" src="https://github.com/zio/zio-metrics-connectors/assets/7413894/3ddeb424-7f3d-401c-bc1e-3ce10b71880d">

Reply:
right, but it's still a constant string that is added to every event. I think it make sense to handle it inside `StatsdEncoder`

Reply:
Makes sense, will change it.

Reply:
Hmm, the thing is, `containerId` is specific to Datadog and does not exist in the Statsd protocol, so it feels a bit weird to move it there. Maybe I could just add a `suffix: Option[String]`?

Reply:
yep ,sounds good to me 👍

Reply:
https://github.com/zio/zio-metrics-connectors/pull/58/commits/0a14a1721f38b5f6b482e096b17fd1b6cf579ac0